### PR TITLE
Add new package xa65

### PR DESCRIPTION
**Pull request by drfiemost**
A 6502/65816 cross assembler, may be useful for VICE or other emulators

### CODE REVIEW

File: mingw-w64-xa65/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=2.3.11
+pkgrel=1
+pkgdesc="6502/65816 cross assembler"
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
+license=('GPL2')
+url="http://www.floodgap.com/retrotech/xa/"
+depends=()
+makedepends=("${MINGW_PACKAGE_PREFIX}-gcc")
```
Comment (Biswa96) [inline mingw-w64-xa65/PKGBUILD:14]:
```suggestion
makedepends=("${MINGW_PACKAGE_PREFIX}-cc")
```

File: mingw-w64-xa65/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+pkgrel=1
+pkgdesc="6502/65816 cross assembler"
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
+license=('GPL2')
+url="http://www.floodgap.com/retrotech/xa/"
+depends=()
+makedepends=("${MINGW_PACKAGE_PREFIX}-cc")
+options=(!strip staticlibs !buildflags)
+source=("https://github.com/fachat/${_realname}/archive/xa-${pkgver}.tar.gz")
```
Comment (lazka) [inline mingw-w64-xa65/PKGBUILD:16]: Why are you using this fork instead of "official" upstream?

Comment (drfiemost) [reply mingw-w64-xa65/PKGBUILD:16]: Main site was down yesterday and I thought this was the official repo, my bad.
And there is also a newer version out...

Comment (drfiemost) [reply mingw-w64-xa65/PKGBUILD]: Fixed and updated

**Pull request merged**